metrics-operator/config/crd/kustomization.yaml

```diff
@@ -4,19 +4,22 @@
 resources:
   - bases/metrics.keptn.sh_keptnmetricsproviders.yaml
   - bases/metrics.keptn.sh_keptnmetrics.yaml
+  - bases/metrics.keptn.sh_analysisdefinitions.yaml
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
   # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
   # patches here are for enabling the conversion webhook for each CRD
   # - patches/webhook_in_keptnmetricsproviders.yaml
   - patches/webhook_in_keptnmetrics.yaml
+  # - patches/webhook_in_analysisdefinitions.yaml
```

Comment:
shouldn't this be enable for the validation webhook to kick in?

Reply:
it's for the conversion webhook, not validation